### ISSUE TITLE
[Suggestion] Add more TFC support

### ISSUE THREAD

**Issue by SpeeeDCraft** [Gregicality/Gregification#4]
**Is your feature request related to a problem? Please describe.**
I want more support for one mod by another. I will also add to the list gradually, since I like to forget: D.
I used a translator, so don't hit me(lul).

- [ ] The hammer from GT should take damage when used in TFC Anvil.
- [ ] Now if you install GT and TFC together, salt water will get a terrible blue texture. We need to fix this.(https://github.com/GregTechCE/GregTech/issues/1561)
- [ ] When welding two ingots on an anvil, you can use a GT hammer and it will receive damage.
- [ ] The GT knife should be able to scrape the skin in the TFC recipe.
- [ ] TFC Salt water or TFC hot water or TFC fresh water, should be used in a coal boiler.
- [ ] It is possible to configure the water configuration if GT is installed together with TFC(true/false). To replace all the water in the recipes of ordinary minecraft with salty/fresh/hot TFC water.
- [ ] And I think the most difficult thing is. So that the generation of GT ores supports the TFC world. This means that the ore from GT will have an overlay not of a stone, but of a TFC stone that is located nearby.
- [ ] With the help of a GT hammer, a wrought iron groove could be driven into a tree, just like with a conventional TFC hammer.(Addon TFC-Tech)
- [ ] With a GT knife, it would be possible to cut the bark of a tree so that latex flows out of it.(Hevea tree)(Addon TFC-Tech)

Perhaps I will add something else here.

**Comment by Rongmario** [comment]
No. 2 is addressed in https://github.com/GregTechCEu/Gregification/pull/13.
